src/plugins/discover/public/application/main/components/sidebar/discover_sidebar.scss

```diff
@@ -70,7 +70,3 @@
     opacity: 1; /* 2 */
   }
 }
-
-.dscSidebar__indexPatternSwitcher {
```

Comment:
it's no longer in use, since the data view switcher went north (up in the unified search bar)